docs/cli/convert.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Convert
+description: Convert SQL migration files to a pgroll migration
+---
+
+## Command
+
+```
+$ pgroll convert --name create_first_table /path/to/migration.sql
+```
+
+This reads the SQL statements from `/path/to/migration.sql` and translates them into a `pgroll` migration.
+The name of the migration is going to be `create_first_table`.
+
+### Read SQL statements from stdin with `pgroll convert`
+
+If file name is not specified `pgroll convert` reads the SQL statements from stdin.
+
+```
+$ cat 'CREATE TABLE my_table(name text);' | pgroll convert --name create_first_table
+```
+
+<Warning>
+The generated pgroll migrations might include `up` and `down` migrations. Those must be filled in manually because
+right now pgroll cannot guess the correct up and down migrations.
```

Comment:
no need for the line break.

Might read better as:

```
The generated `pgroll` migrations might include `up` and `down` data migrations. Those must be filled in manually because currently `pgroll` is unable to infer correct up and down migrations.
```